packages/web3-common/test/unit/formatters.test.ts

```diff
@@ -0,0 +1,674 @@
+import * as utils from 'web3-utils';
+
+import { PREDEFINED_BLOCK_NUMBERS } from '../../src/constants';
+import {
+	inputAddressFormatter,
+	inputBlockNumberFormatter,
+	inputDefaultBlockNumberFormatter,
+	inputPostFormatter,
+	isPredefinedBlockNumber,
+	outputBigIntegerFormatter,
+	outputBlockFormatter,
+	outputLogFormatter,
+	outputPostFormatter,
+	outputProofFormatter,
+	outputSyncingFormatter,
+	outputTransactionReceiptFormatter,
+	txInputOptionsFormatter,
+} from '../../src/formatters';
+
+import * as formatters from '../../src/formatters';
+
+// TODO: The issue with export * causing issue with jest.spyOn
+// have to hack this way until find resolutions
+jest.mock('web3-utils', () => ({
+	__esModule: true,
+	...jest.requireActual('web3-utils'),
+}));
```

Comment:
```suggestion
jest.mock('web3-utils');
```

I've tested this works locally, opened a [PR](https://github.com/ChainSafe/web3.js/pull/4460) but forgot the tests wouldn't run there